setup.py

```diff
@@ -24,6 +24,6 @@
     packages=[
         "multipoint",
     ],
-    install_requires=["numpy>=1.16", "mpi4py>=3.0"],
+    install_requires=["numpy>=1.16", "mpi4py>=3.0", "mdolab-baseclasses>=1.3"],
```

Comment:
Is the plan to release v1.3 today?

Reply:
I was contemplating that since we have several new features and functions there. Perhaps we want to wait a bit more for a few other changes to settle in and then bump the minor release. I will change this to v1.2.4 since this is the first version at has the `Error` class.